cmd/mt-fakemetrics/cmd/parrot.go

```diff
@@ -0,0 +1,102 @@
+package cmd
```

Comment:
this tool is beyond just sending fake metrics.
as this will send fake data into a MT cluster and then query it back, it's a "real" MT testing tool, so i would put it in <root>/cmd/mt-parrot. you then also don't have to do the cobra stuff.

Reply:
to be clear, we don't want a tool that just sends the test data.
we want a tool that can execute the entire test.
it can use the fakemetrics library to do its work.

Reply:
understood, was planning on adding that in next, just sanity checking this portion before building on it

Reply:
also i realize this is a departure from the suggestion i have given you before (which was a new mode for fakemetrics tool + a separate query/test tool), but it seems to me doing both in one tool is simpler.